lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -3872,6 +3872,38 @@ class DecomposeAtenNewEmptyStridedOp
 };
 } // namespace
 
+namespace {
+class DecomposePrimsSqueezeOp : public OpRewritePattern<PrimsSqueezeOp> {
+public:
+  using OpRewritePattern::OpRewritePattern;
+  LogicalResult matchAndRewrite(PrimsSqueezeOp op,
+                                PatternRewriter &rewriter) const override {
+    Location loc = op.getLoc();
+    Value input = op.getA();
+
+    SmallVector<int64_t> dimensions;
+    if (!matchPattern(op.getDimensions(),
+                      m_TorchListOfConstantInts(dimensions)))
+      return rewriter.notifyMatchFailure(
+          op, "all dimensions must be constant ints");
+
+    if (dimensions.size() > 1)
+      return rewriter.notifyMatchFailure(
+          op, "unimplemented: prims.squeeze op supported only for dimensions "
+              "list containing single value");
```

Comment:
Having full support does not require that much more work, so we should just add it. You just need to sort `dimensions in decreasing order and call `AtenSqueezeDimOp` for each dimension. (Do remember to check that `a.size(dim) == 1` for each `dim`.